services/src/datasets/external/netcdfcf/error.rs

```diff
@@ -0,0 +1,92 @@
+use gdal::errors::GdalError;
+use geoengine_datatypes::dataset::DatasetProviderId;
+use snafu::Snafu;
+
+// TODO: `Clone` and `PartialEq`
```

Comment:
why is it needed later?

Reply:
It actually sucks if you cannot do this with errors, especially for testing